lively-server/auth.js

```diff
@@ -0,0 +1,89 @@
+import jwt from 'jsonwebtoken';
+import _ from 'lodash';
+import bcrypt from 'bcrypt';
+import models from './models/index';
+export const createTokens = async (user, secret) => {
+    const token = jwt.sign(
+        {
+            user: _.pick(user, ['id'])
+        },
+        secret,
+        {
+            expiresIn: '1h'
+        }
+    );
+    const refreshToken = jwt.sign(
+        {
+            user: _.pick(user, 'id'),
+        },
+        secret,
+        {
+            expiresIn: '7d',
+        }
+    )
+    return {refreshToken, token}
+}
+
+export const tryLogin = async (email, password, models, SECRET) => {
+    const user = await models.User.findOne({ where: { email }, raw: true});
+    if (!user) {return {
+        ok: false,
+        errors: [{path: 'email', message: 'Invalid email'}],
+    }}
+    const valid = await bcrypt.compare(password, user.password);
```

Comment:
this should 100% be handled by an actual auth library. try using one, it will probably introduce some constraints that weren't previously accounted for, and it's good practice to pick up new libraries regularly anyways